app/views/browse/mobile-clients.html

```diff
@@ -60,6 +60,23 @@ <h3>Mobile Client Config</h3>
                 </div>
               </div>
             </uib-tab>
+            <uib-tab active="selectedTab.mobileServices" class="mobile-services">
+              <uib-tab-heading>Mobile Services</uib-tab-heading>
+              <div class="overview" ng-if="ctrl.serviceInstances | size">
+                <div class="list-pf">
+                  <mobile-service-instance-row
+                    ng-repeat="serviceInstance in ctrl.serviceInstances track by (serviceInstance | uid)"
+                    api-object="serviceInstance" 
+                    binding="ctrl.bindings[serviceInstance.metadata.name]"
+                    service-class="ctrl.serviceClasses[serviceInstance.spec.clusterServiceClassRef.name]">
+                  </mobile-service-instance-row>
+                </div>
+              </div>
+              <div class="note" ng-if="!(ctrl.serviceInstances | size)">
+                <p>Provision a mobile service to integrate it with your mobile client.</p>
+                <a class="btn btn-primary btn-lg" ng-href="project/{{ctrl.projectName}}/catalog?category=mobile&subcategory=services">Browse Catalog</a>
```

Comment:
they prefer to use navigate e.g.  https://github.com/aerogear/origin-web-console/commit/c151097753861c751b81103f2893bd2ceef29644#diff-4cc5d017ab8a77ecfad04db4f53e2066R66